doc/user_guide/dynamo0p3.rst

```diff
@@ -1527,6 +1540,24 @@ function space ``Wtheta`` is given in ``examples/lfric/eg9``, with the
 fields. This example also demonstrates how to only colour loops over
 continuous function spaces when transformations are applied.
 
+The previously mentioned **FEM order** can be specified in the horizontal
```

Comment:
This is a very well written paragraph, and I only have minor remarks on it (see inline). I would suggest moving it up, though, after reading the generated documentation. I think it would follow more naturally between the description of the specific function spaces (`Wchi` being the last of them) and generic `ANY_*` spaces (the paragraph that starts with "In addition to the specific function space metadata, there are also three generic function space..."). I think that the "FEM order" would not need to be in bold then as it is closer to the original mention.